styles.css

```diff
@@ -197,6 +197,243 @@ header {
+  align-items: center;
+  background-color: #6070ff;
+  color: #ebebff;
+  border-top-left-radius: 80px;
+}
+
+.web-languages-card {
+  margin: 5px;
+  padding: 20px;
+  padding-top: 20px;
+  padding-left: 10px;
+  background: #f7f7f9;
+  size: 20px;
+}
+
+.languages {
+  display: flex;
+  flex-direction: column;
+}
+
+.web-languages {
+  display: flex;
+}
+
+.about-introduction {
+  color: #091e42;
+  font-weight: bolder;
+  font-size: 30px;
+}
+
+.technologies-link {
+  padding-top: 15px;
+  padding-bottom: 15px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.languages-title {
+  display: flex;
+  justify-content: space-between;
+}
+
+h4 {
+  color: #7f8cff;
+  font-size: 10px;
+  font-weight: 500;
+}
+
+.contact-me {
+  color: #fff;
+  font-weight: 700;
+  font-size: 25px;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  width: 50%;
+  margin: 40px 0;
+  gap: 10px;
+}
+
+form input {
+  padding: 10px;
+  border-radius: 10px;
+}
+
+.container-contact {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 50%;
+  margin: 0 auto;
+  align-items: center;
+}
+
+.submit {
+  color: #396df2;
+  border: 1px solid #396df2;
+  border-radius: 6px;
+  width: 40%;
+  padding: 10px;
+  margin-left: 100px;
+}
+
+.works {
+  display: grid;
+  grid-template-columns: 1fr;
+  margin: 0 auto;
+  padding: 0 20px;
+  gap: 8rem;
+}
+
+.work-item {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  gap: 2rem;
+  background-color: white;
+  padding: 1rem;
+  border-radius: 6px;
+}
+
+.work-item:first-child {
+  margin-top: 30px;
+}
+
+.project-tags {
+  margin-bottom: 30px;
+}
+
+.project-tags ul {
+  display: flex;
+  flex-direction: row;
+  list-style: none;
+  padding-left: 0;
+}
+
+.container-technologies li {
+  border-bottom: 1px solid #dfe1e6;
+}
+
+.project-tags ul li {
+  margin-right: 8px;
+  padding-right: 0;
+  padding-left: 0;
+}
+
+.container-skill ul {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  margin-bottom: 20px;
+  list-style: none;
+  padding-left: 0;
+}
+
+.container-skill ul li {
+  margin-right: 10px;
+}
+
+.project-description {
+  text-align: left;
+  font-weight: 400;
+  size: 16px;
+  margin-top: 20px;
+  margin-bottom: 30px;
+}
+
+.project-headline {
+  color: #091e42;
+  font-weight: 900;
+  font-size: 24px;
+  text-align: left;
+}
+
+.fa-circle {
+  font-size: smaller;
+  font-weight: 200;
+  color: #c1c7d0;
+}
+
+.skill-button {
+  color: #396df2;
+  border: 1px transparent;
+  border-radius: 6px;
+  width: 100%;
+  padding: 5px;
+  margin-right: 30px;
+  text-align: center;
+  font-size: 10px;
+}
+
+.see-project {
+  color: #396df2;
+  border: 1px solid #396df2;
+  border-radius: 6px;
+  padding: 10px 20px;
+  margin-top: 20px;
+  background-color: white;
+}
+
+.see-project:hover {
+  background-color: #6070ff;
+  color: white;
+  cursor: pointer;
+}
+
+.work-item-detail {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
```

Comment:
- [ ] [OPTIONAL] Great job 👍 .Please make sure your design matches the Figma design as much as possible because it is a good practice and makes one a better developer 🙏 
- You can try removing the margin and padding.

Figma Design | Your Design
------------- | -------------
![Figma Design](https://user-images.githubusercontent.com/74814780/176548335-22322a9d-f525-4105-8775-7ed3374f95fa.png)  |  ![work](https://user-images.githubusercontent.com/74814780/176548123-659b38fc-6c6b-4d36-ae39-ca91f768b8a9.png)